iohub/pyramid.py

```diff
@@ -0,0 +1,46 @@
+import math as m
```

Comment:
This alias is only used once. Also abbreviating stdlib is not much of a standard practice.